### PR TITLE
warnings/typos on 2.1

### PR DESCRIPTION
**Pull request by hubertp-lshift**

### CODE REVIEW

File: docs/user/smv_input.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ object acct_demo extends SmvCsvFile("accounts/acct_demo.csv")
 ### Python
 ```Python
 # In file src/main/python/stage1/inputdata.py
-class acct_demo(SmvCsvFile):
+class acct_demo(PySmvCsvFile):
```
Comment (ninjapapa) [inline docs/user/smv_input.md]: Should be `SmvCsvFile`

File: src/main/scala/org/tresamigos/smv/dqm/DQMState.scala [hunk 2]
```diff
@@ -25,15 +30,15 @@ import scala.collection.mutable.MutableList
...
     fixNames: Seq[String]
   ) extends Serializable {
 
-  private val recordCounter: Accumulator[Long] = sc.accumulator(0l)
-  private val parserLogger = new RejectLogger(sc, 10)
-  private val fixCounters: Map[String, Accumulator[Int]] = fixNames.map{n => (n, sc.accumulator(0))}.toMap
-  private val ruleLoggers: Map[String, RejectLogger] = ruleNames.map{n => (n, new RejectLogger(sc, 10))}.toMap
+  private val recordCounter: LongAccumulator = sc.longAccumulator
+  private val parserLogger = new RejectLogger(sc, 10, "parser")
+  private val fixCounters: Map[String, IntAccumulator] = fixNames.map{ n => (n, new IntAccumulator)}.toMap
```
Comment (ninjapapa) [inline src/main/scala/org/tresamigos/smv/dqm/DQMState.scala]: Use the builtin LongAccumulater for `fixCounters` should be fine.

File: docs/user/smv_python_progress.md [hunk 3]
```diff
@@ -6,6 +6,7 @@
   * [x] SmvPyModule
   * [x] SmvPyOutput
   * [ ] SmvPyModuleLink
+  * [ ] SmvPyMultiCsvFile
```
Comment (jacobdr) [inline docs/user/smv_python_progress.md]: I think that both items -- SmvPyModuleLink and SmvPyMultiCsvFile  --  have actually been implemented. @laneb

**Comment by AliTajeldin** [comment]
@hubertp-lshift : Looks real good. I especially like the new `findDataSetInRepo` implementation.  merging shortly.

File: src/main/scala/org/tresamigos/smv/dqm/DQMState.scala (hunk 2 above)
Comment (hubertp-lshift) [inline src/main/scala/org/tresamigos/smv/dqm/DQMState.scala]: Right, I pretty much inferred their kind from the initial value. I can fix that shortly.

**Comment by hubertp-lshift** [comment]
@jacobdr Yeah, I haven't yet figured out why there are memory leaks. Maybe they were there from the beginning except that because the build was Maven-based you wouldn't see it (clean build restarts). I'm just speculating here.

File: docs/user/smv_python_progress.md (hunk 3 above)
Comment (hubertp-lshift) [inline docs/user/smv_python_progress.md]: I just checked, that's correct, indeed. Some aliases were put in place.

**Pull request merged**